### PR TITLE
Added function to mark a Beer as drunk

### PR DESCRIPTION
**Pull request by tistatos**
The drink slug is saved in cookie, if we have that cookie, that beer has been drink,
No remove feature at the moment though

Also added a quick "last updated at" in the sortiment

### CODE REVIEW

File: app/controllers/drinks_controller.rb [hunk 1]
```diff
@@ -1,20 +1,22 @@
 class DrinksController < ApplicationController
-  before_action :set_drink, only: [:show, :edit, :update, :destroy, :change_stock]
-  before_action :authenticate_user!, except: [:show]
+  before_action :set_drink, only: [:show, :edit, :update, :destroy, :change_stock, :cookie]
```
Comment (klaseskilson) [inline app/controllers/drinks_controller.rb]: We should switch to `except` instead of `only` here.

File: app/controllers/drinks_controller.rb [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+  before_action :authenticate_user!, except: [:show, :cookie]
   protect_from_forgery except: [:change_stock]
-  authorize_resource
+  authorize_resource except: [:cookie]
```
Comment (klaseskilson) [inline app/controllers/drinks_controller.rb]: Why shouldn't we authorize this here?

File: app/controllers/drinks_controller.rb [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
     @drinks = Drink.all
     @drink_types = DrinkType.all
+
   end
 
   # GET /sortiment/:id/dryck/1
   # GET /sortiment/:id/dryck/1.json
   def show
     @api_info = BREWERY.search.beers(q: @drink.brewery + ' ' + @drink.name).first
+    @have_drank = cookies[@drink.slug];
```
Comment (klaseskilson) [inline app/controllers/drinks_controller.rb]: `;`!

File: config/routes.rb [hunk 4]
```diff
@@ -20,7 +20,11 @@
   end
 
   resources :drink_types, path: '/sortiment', as: :stock, only: [:index, :show] do
-    resources :drinks, path: '/', only: [:show]
+    resources :drinks, path: '/', only: [:show] do
+      member do
```
Comment (klaseskilson) [inline config/routes.rb]: why not make a separate route for this?

File: db/seeds.rb [hunk 5]
```diff
@@ -1,6 +1,24 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
 
-User.create(email: 'klas.eskilson@gmail.com', admin: true)
-User.create(email: 'englund92@gmail.com', admin: true)
-User.create(email: 'daniel@ronnkvist.com', admin: true)
+#User.create(email: 'klas.eskilson@gmail.com', admin: true)
```
Comment (klaseskilson) [inline db/seeds.rb]: Great idea to get rid of these. I'd say they can be remove and not just commented out.

File: app/controllers/drinks_controller.rb (hunk 2 above)
Comment (tistatos) [inline app/controllers/drinks_controller.rb]: if I don't use except: here I get a `You are not authorized to access this page.`-message from cancan

File: app/controllers/drinks_controller.rb (hunk 3 above)
Comment (tistatos) [inline app/controllers/drinks_controller.rb]: kk, will fix

**Comment by tistatos** [comment]
BUMP :smile:

**Comment by tistatos** [comment]
Beverages can now also be unmarked as drunk

**Comment by tistatos** [comment]
här gåre snabbt!

File: config/routes.rb [hunk 6]
```diff
@@ -21,12 +21,15 @@
...
 
+  get '/sortiment/:id/drunk' => 'drinks#cookie', as: :drunk
+
   resources :drink_types, path: '/sortiment', as: :stock, only: [:index, :show] do
     resources :drinks, path: '/', only: [:show]
   end
 
   resources :news, path: 'nyheter', as: :public_news, only: [:show]
 
+
```
Comment (klaseskilson) [inline config/routes.rb]: dog

**Comment by klaseskilson** [comment]
I'd say that the `limit` params in `db/schema.rb` are good (since they actually declare the limit explicitly instead of under the hood). I'm not sure why you removed them.

Other than that it looks perfect! :beer: :beers: :beer: :beers: :beer: :beers: Great bug fix and nice touch with the "last updated at"-thingy.

**Comment by tistatos** [comment]
I merge-fixed the `limit` but my rake removed them...

**Comment by klaseskilson** [comment]
All right! Since my rake produced the `limit`s I assumed that it was a new standard. My bad! It's all looking good to me then!

**Pull request merged**